<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TomatoSetClass</class>
 <widget class="QDialog" name="TomatoSetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置番茄</string>
  </property>
  <property name="windowIcon">
   <iconset resource="QTTODO.qrc">
    <normaloff>:/QTTODO/tomato_Icon~1.ico</normaloff>:/QTTODO/tomato_Icon~1.ico</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,0">
      <item row="5" column="1">
       <widget class="QCheckBox" name="isForwardCheck">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nameEdit"/>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="restspinBox">
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="tomatospinBox">
        <property name="maximum">
         <number>40</number>
        </property>
        <property name="value">
         <number>40</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>执行次数</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>休息时间（min）</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>番茄时间（min）</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="performspinBox">
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>是否正向计时</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTimeEdit" name="timeEdit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>番茄名字</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>开始时间</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="YesButton">
        <property name="text">
         <string>确认</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="1">
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>周二</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QRadioButton" name="radioButton_6">
        <property name="text">
         <string>周六</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="radioButton_4">
        <property name="text">
         <string>周四</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="radioButton_3">
        <property name="text">
         <string>周三</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QRadioButton" name="radioButton_5">
        <property name="text">
         <string>周五</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>周一</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QRadioButton" name="radioButton_7">
        <property name="text">
         <string>周日</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>设置周几</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QTTODO.qrc"/>
 </resources>
 <connections/>
</ui>
